types/chenyfan__cache-db/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for @chenyfan/cache-db 0.0
+// Project: https://www.npmjs.com/package/@chenyfan/cache-db
+// Definitions by: AHdark <https://github.com/AH-dark>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace CacheDB {
+    type Config = "json" | "arrayBuffer" | "blob" | "text" | string;
```

Comment:
Lines 17 and 41 of `index.js` show config as an object with key `type`: <https://unpkg.com/browse/@chenyfan/cache-db@0.0.4/index.js>

```suggestion
    interface Config {
        type: "json" | "arrayBuffer" | "blob" | "text" | string;
    }
```